charts/optimize-live/templates/deployment.yaml

```diff
@@ -99,6 +95,14 @@ spec:
             {{- toYaml .Values.readinessProbe | nindent 12 }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+          - name: config
+            mountPath: "/config"
```

Comment:
@bradbeam mentioned use subpath and rm /config, ref: https://github.com/gramLabs/optimize-live-controller/blob/main/pkg/components/tsdb/deployment.go#L93-L97